address_book/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.17
+
+WORKDIR /workdir
+ADD . .
+RUN go mod download
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o bin/app cmd/server/main.go
+
+EXPOSE 8080
+
+ENTRYPOINT ["/workdir/bin/app"]
+
+#FROM golang
```

Comment:
Please look at these commented lines again, they have a very good tip on the proper approach on building golang images.

Reply:
i added multistage build. Done (Also update docker image on DockerHub)